index.html

```diff
@@ -3042,26 +2948,35 @@ <h3>Revocation</h3>
       <section>
         <h3>Fitness for Purpose</h3>
 
-        <ul>
-          <li>The custom properties in the <a>credential</a> are
-          appropriate for the <a>verifier</a>'s purpose. For example if a
-          verifier needs to determine that a <a>subject</a> is older than 21
-          years of age, they may rely on claims of a specific
-          <code>birthdate</code> or abstract properties such as
-          <code>ageOver</code>.</li>
-          <li>The <a>issuer</a> is trusted by the <a>verifier</a> to make the
-          claims at hand. For example, a franchised Fast Food restaurant
-          location will trust discount coupon claims made by the
-          corporate headquarters of the franchise.</li>
-          <li>Policy information expressed by the <a>issuer</a>
-          in the <a>verifiable credential</a> SHOULD be respected by holders 
-		  and verifiers unless they accept the liability of ignoring the policy.
-
-		  For example, the issuer may wish to limit the use
-          of the credential to specific <a>verifiers</a>, certain <a>holder</a>
-          age ranges, or during certain dates.</li>
-          <li>All policy information expressed by the <a>holder</a>
-          in the <a>verifiable presentation</a> SHOULD be respected by verifiers
+        <p>
+The custom properties in the <a>credential</a> are
+appropriate for the <a>verifier</a>'s purpose. For example if a
+verifier needs to determine that a <a>subject</a> is older than 21
+years of age, they may rely on claims of a specific
+<code>birthdate</code> or abstract properties such as
+<code>ageOver</code>.</p>
+
+          <p>
+The <a>issuer</a> is trusted by the <a>verifier</a> to make the
+claims at hand. For example, a franchised Fast Food restaurant
+location will trust discount coupon claims made by the
+corporate headquarters of the franchise.</li>
+Policy information expressed by the <a>issuer</a>
+in the <a>verifiable credential</a> SHOULD be respected by holders 
+  and verifiers unless they accept the liability of ignoring the policy.
+</p>
+          <p>
+        <p>
+The <a>issuer</a> is trusted by the <a>verifier</a> to make the claims at
+hand. For example, a franchised fast food restaurant location will trust
+discount coupon claims made by the corporate headquarters of the franchise.
```

Comment:
This looks like duplication of the paragraph above

Reply:
Yep. Thanks.